src/Lumino/index.js

```diff
@@ -82,8 +83,9 @@ const Lumino = () => {
   const getRNSOptions = () => {
     return {
       networkId: this.chainId,
+      // TODO: Implement this
       contractAddresses: {
-        registry: rifConfig.rns.contracts.rns,
+        registry: AddressZero,
```

Comment:
Dont understand this, why removing the rifConfig param?

Reply:
This was undefined, rifConfig was a .ENV variable, so the SDK doesn't hold any reference to it.
I will fix this with the configParams.registryAddress

Reply:
Fixed with a new commit, the address is now in luminoConfig